js/index.js

```diff
@@ -1,14 +1,18 @@
+// 1. create copyright element in footer
+
 const today = new Date();
 const thisYear = today.getFullYear();
 
-const footer = document.querySelector('footer');
+const footer = document.querySelector('footer .copyright');
 
 const copyright = document.createElement('p');
-copyright.innerHTML = `&copy; ${thisYear} Irina Postnova`;
+copyright.innerHTML = `&copy; ${thisYear} Irina Postnova. <br> For educational purposes.`;
```

Comment:
This is a super tiny nitpick, but since `<div>` are block elements, wrapping content in divs can be a more semantic way to get this line break without using `<br>`

It would also allow you to assign a classname and apply specific styling to each line of the message if you want

```suggestion
copyright.innerHTML = `<div>&copy; ${thisYear} Irina Postnova.</div><div>For educational purposes.</div>`;
```